.storybook/main.js

```diff
@@ -1,6 +1,8 @@
 const path = require('path');
 const fs = require('fs');
 const { TsconfigPathsPlugin } = require('tsconfig-paths-webpack-plugin');
+const { getAllPackageInfo, isConvergedPackage } = require('@fluentui/scripts/monorepo');
```

Comment:
can we avoid creeping scripts low level api's to storybook ? It would be great if we could encapsulate this logic somewhere

Reply:
but where to encapsulate it ? `main.js` is the entrypoint into storybook

Reply:
not sure TBH, somewhere in scripts or so?, lets keep it as is for now

Reply:
> assuming the primary goal 

primary goal is to have standardized codebase that follows right boundaries, which goes hand in hand with extraction/encapsulation etc